src/libs/tagging.c

```diff
@@ -102,7 +99,7 @@ const char **views(dt_lib_module_t *self)
   static const char *v1[] = {"lighttable", "darkroom", "map", "tethering", NULL};
   static const char *v2[] = {"lighttable", "map", "tethering", NULL};
 
-  if(dt_conf_get_bool("plugins/darkroom/tagging/visible"))
+  if(dt_conf_get_bool("plugins/lighttable/tagging/visible"))
```

Comment:
Looks a wrong change. Can you explain? lt only view is v2.

Reply:
correct I've made confusion with the new flags. Ok.